packages/create-cloudflare/e2e-tests/pages.test.ts

```diff
@@ -26,25 +26,31 @@ describe("E2E", () => {
 		}
 	});
 
-	const runCli = async (framework: string) => {
+	const runCli = async (framework: string, args: string[] = []) => {
 		const projectPath = join(dummyPath, "test");
-		const argv = [
+
+		let argv = [
 			projectPath,
 			"--type",
 			"webFramework",
 			"--framework",
 			framework,
 			"--no-deploy",
-			"--no-git",
-			"--wrangler-defaults",
 		];
 
+		if (args.length > 0) {
+			argv = [...argv, ...args];
```

Comment:
```suggestion
			argv.push(...args);
```

And similarly below, which means you can leave `argv` as a `const` above.